<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="FTR3COF93BK8A76" name="TestCases.WFA_POTS_Dial_Tone_2" tests="2" failures="1" timestamp="2022-03-24T04:33:03 EDT" time="29.928" errors="0">
  <testcase name="DT_23" time="23.192" classname="TestCases.WFA_POTS_Dial_Tone_2">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [106] line_end; expected [I would like to assist you in determining if the trouble is at your location or in your telephone equipment. Are you interested in hearing the trouble-shooting steps?If the customer is interested, select &#039;Willing to check&#039;] but found [I would like to assist you in determining if the trouble is atyour location or in your telephone equipment. Are youinterested in hearing the trouble-shooting steps?If the customer is interested, select &#039;Willing to check&#039;]gui_start  gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
@@line_start [106] line_end; expected [I would like to assist you in determining if the trouble is at your location or in your telephone equipment. Are you interested in hearing the trouble-shooting steps?If the customer is interested, select 'Willing to check'] but found [I would like to assist you in determining if the trouble is atyour location or in your telephone equipment. Are youinterested in hearing the trouble-shooting steps?If the customer is interested, select 'Willing to check']gui_start  gui_end
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at TestCases.WFA_POTS_Dial_Tone_2.DT_23(WFA_POTS_Dial_Tone_2.java:141)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- DT_23 -->
  <system-out/>
  <testcase name="DT_24" time="6.736" classname="TestCases.WFA_POTS_Dial_Tone_2"/>
  <system-out/>
</testsuite> <!-- TestCases.WFA_POTS_Dial_Tone_2 -->
